docs/flink-writes.md

```diff
@@ -69,7 +69,7 @@ Iceberg supports `UPSERT` based on the primary key when writing data into v2 tab
 
 ```sql
 CREATE TABLE `hive_catalog`.`default`.`sample` (
-    `id`  INT UNIQUE COMMENT 'unique id',
+    `id`  INT COMMENT 'unique id',
```

Comment:
nit: Could we remove the extra space after the column name too?

Reply:
Thanks for reviewing. Have removed the extra space.